client/src/app/site/polls/components/base-poll-detail.component.ts

```diff
@@ -213,6 +209,24 @@ export abstract class BasePollDetailComponent extends BaseModelContextComponent
                 {
                     idField: 'content_object_id'
                 },
+                /**
+                 * Get "single votes" as users (short names + vote weight)
+                 * should be ViewPoll.voted ?
+                 * Nothing works at all.
+                 * 1. Get the candidates from
+                 */
+                {
+                    idField: 'voted_ids' // what was expected: not working
+                },
+                // {
+                //     idField: 'poll_voted' // not working
+                // },
+                // {
+                //     idField: {
+                //         templateIdField: 'poll_voted_$_ids',
+                //         templateValue: this.activeMeetingId?.toString()
+                //     }
+                // },
```

Comment:
This is not getting me anywhere. The new request system still proves to be a foot gun.
Wither the relations are just wrong and "voted" is not what I expect it to be or requesting fields from the poll works different than all other models.

@FinnStutzenstein @GabrielInTheWorld

Reply:
I have committed an update to this PR. With this update you get the `voted`-users. (`poll.voted`)
You can read through this update and ask me, if you have questions.